### PR TITLE
Add Bert_large

### PR DESCRIPTION
**Pull request by davidberard98**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #1322
* __->__ #1321

Useful for testing FSDP, where we want to test on large models.

Differential Revision: [D41478058](https://our.internmc.facebook.com/intern/diff/D41478058)

### CODE REVIEW

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D41478058).

File: torchbenchmark/models/hf_Bert_large/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+devices:
+  NVIDIA A100-SXM4-40GB:
+    eval_batch_size: 4
+eval_benchmark: false
+eval_deterministic: false
+eval_nograd: true
+not_implemented:
+- jit: true
+train_benchmark: false
```
Comment (wconstab) [inline torchbenchmark/models/hf_Bert_large/metadata.yaml:9]: what does this mean?

**Review by wconstab — APPROVED** [review]

File: torchbenchmark/models/hf_Bert_large/metadata.yaml (hunk 1 above)
Comment (davidberard98) [reply torchbenchmark/models/hf_Bert_large/metadata.yaml:9]: This is config for CI; I just copied these from hf_Bert/metadata.yaml (and reduced batch size). Not sure exactly what train_benchmark does, (@xuzhao9 do you know? I did some grepping but still couldn't tell what it was doing)

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D41478058).

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D41478058).

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D41478058).

**Pull request closed without merging**